values.yaml

```diff
@@ -139,6 +139,8 @@ workers:
   # Persistence.enabled must be set to false to use KEDA.
   keda:
     enabled: false
+    namespaceLabels: |-
+      name: astronomer
```

Comment:
@dimberman this namespace is not guaranteed each time, only when using our terraform.

Also, now that this might be used more generally by the community, it feels like we shouldn't have this pre-populated. Could we pull the default value out here and pass from Houston?

Reply:
Also if you’re going to change this, I prefer dict vs block string and toYaml vs tpl

Reply:
Make sure to test empty dict

Reply:
sure